samples/WebApi/README.md

```diff
@@ -0,0 +1,115 @@
+# Description
+This is a sample application, which uses CoreRT to compile a .NET Core Web API sample.
+TODO WIP - Might not work on Linix
+
+# Building a WebAPI app with CoreRT
+
+## Install the .NET Core SDK
+CoreRT is and AOT-optimized .NET Core runtime. If you're new to .NET Core make sure to visit the [official starting page](http://dotnet.github.io). It will guide you through installing pre-requisites and building your first app.
```

Comment:
Typo: should be "an AOT-optimized .NET Core runtime".

Reply:
Good catch, thanks!